src/modules/esl-toggleable/core/esl-toggleable.ts

```diff
@@ -196,9 +196,17 @@ export class ESLToggleable extends ESLBaseElement {
     return this;
   }
 
+  /**
+   * Actions to execute on show task when toggleable is in the open state.
+   * @returns whether the show task should be run to the end.
+   */
+  protected onParamsUpdate(params: ESLToggleableActionParams): boolean | void {
+    this.activator = params.activator;
+  }
+
   /** Actual show task to execute by toggleable task manger ({@link DelayedTask} out of the box) */
   protected showTask(params: ESLToggleableActionParams): void {
-    if (!params.force && this.open) return;
+    if (!params.force && this.open && !this.onParamsUpdate(params)) return;
```

Comment:
Not sure about the case with `force=true` in terms of whether should we skip the onParmsUpdate execution